### PR TITLE
1st_negative_no_in_every_window

### PR DESCRIPTION
**Pull request by bitaashna**
### Have you read the [Contributing Guidelines on Pull Requests]
Yes,I have read it

### Description
I have implemented the code to find maximum sum subarray of size k with the help of sliding window in cpp .

### Checklist

- [x]  I've read the contribution guidelines.
- [x]  I've checked the issue list before deciding what to submit.
- [x]  I've edited the README.md and link to my code.

### Related Issues or Pull Requests
#2304

### CODE REVIEW

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -263,6 +263,9 @@ _add list here_
 - [Minimum number of insertions and deletions](dp/min_ins_del.cpp)
 - [Unbounded Knapsack](dp/unbounded_knapsack.cpp)
 
+## Sliding Window
+- [First Negative Number in Every window of size k](sliding_window/1st_negative_no_in_every_window_of_size_k.cpp)
```
Comment (HarshCasper) [inline C-Plus-Plus/README.md]: Move this to `cp`

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Indent and format the code properly

**Review by bitaashna — COMMENTED** [review]
@HarshCasper I have made the changes.Please review!!

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Format the Code using a formatter

**Review by bitaashna — COMMENTED** [review]
@HarshCasper I have formatted the code.Please check!

File: C-Plus-Plus/README.md [hunk 2]
```diff
@@ -68,7 +68,8 @@
 - [Conversion of Binary Number to Decimal Number](cp/Conversion_from_Binary_to_Decimal.cpp)
 - [Painting Fence Algorithm](cp/Paint_Fence.cpp)
 - [All String permutation by using recursion](cp/recursion_string_permutation.cpp)
-- [Peak element](cp/Peak_element.cpp)
+- [First Negative Number in Every window of size k](cp/1st_negative_no_in_every_window_of_size_k.cpp)
+- [Peak element](cp/Peak_element.cpp]
```
Comment (HarshCasper) [inline C-Plus-Plus/README.md]: Not done properly

File: C-Plus-Plus/1st_negative_no_in_every_window_of_size_k.cpp [hunk 3]
```diff
@@ -0,0 +1,82 @@
+/*
+Problem Statement:
+An array and a positive integer window_size(k) is given,we need to find 1st negative integer for each and every window(contiguous subarray) of size k
+
+Input format:-
+ size(n)
+ array elements
+ window_size(k)
+Output format:-
+ size-window_size+1 elements
```
Comment (HarshCasper) [inline C-Plus-Plus/1st_negative_no_in_every_window_of_size_k.cpp]: Make it more intuitive

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by bitaashna — COMMENTED** [review]
@HarshCasper please review!

**Comment by bitaashna** [comment]
@HarshCasper Sorry for the inconvenience but I will be creating a fresh PR..as I deleted the forked repo and forked it again.
So to avoid any issue, I will be opening a new PR

**Pull request closed without merging**